build/config/common-settings.json

```diff
@@ -83,6 +83,7 @@
    "devtools.overlay": false,
    "devtools.overlay.system": false,
    "devtools.eventlooplag.threshold": 100,
+   "devtools.reflowduration.threshold": 16,
```

Comment:
Nit: Most of the HUD settings start with the `hud.` prefix instead of `devtools.`.

Add-on nit: Also, the duration-threshold you're creating here is for all the reflow-related metrics. Please call it something like `hud.reflows.threshold`. (I know it's inconsistent with `devtools.eventlooplag.threshold`, but I'd love to rename that as well eventually).